service/access/server.go

```diff
@@ -448,20 +450,20 @@ func (s *Server) ExecuteScriptAtBlockHeight(_ context.Context, in *access.Execut
 	for _, arg := range in.Arguments {
 		val, err := json.Decode(nil, arg)
 		if err != nil {
-			return nil, fmt.Errorf("could not decode script argument: %w", err)
+			return nil, status.Errorf(codes.Internal, "could not decode script argument: %v", err)
```

Comment:
Since these are passed in from the client, an error probably means the client passed invalid data
```suggestion
			return nil, status.Errorf(codes.InvalidArgument, "could not decode script argument: %v", err)
```